packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx

```diff
@@ -33,18 +34,15 @@ export const ArrowIndicatorCell = ({
     },
   } = cell;
   const { palette, shape, spacing } = useTheme();
-  const colorOnUp = greenOnArrowUp ? palette.primary.light : palette.error.main;
-  const colorOnDown = greenOnArrowUp
+  const arrowColor = tooRisky
     ? palette.error.main
+    : greenOnArrowUp
+    ? colors.orange
     : palette.primary.light;
-  let backgroundColor = palette.info.light;
-  // Don't show background highlight on default
-  if (arrowUp === null) backgroundColor = palette.background.default;
-  if (arrowUp === true && greenOnArrowUp === false)
-    backgroundColor = palette.error.light;
-  if (arrowUp === false && greenOnArrowUp === true)
-    backgroundColor = palette.error.light;
-  const valueTextColor = isNegative
+
+  const valueTextColor = tooRisky
```

Comment:
The nested ternery here is hard to read. What about  `tooRisky || isNegative ? palette.error.main : ...`

Reply:
Can do